AnomalyDetection.py

```diff
@@ -85,27 +83,53 @@ def DetectAnomalies(data, windowSize, levels=1, numTopResults=None, col=0):
 
     #Visualize the outputs
     timestamps = data.index
-    print('{0: <45}Anomaly Score'.format('Time Interval'))
+    if noisy:
+        print('{0: <45}Anomaly Score'.format('Time Interval'))
     low = results[min(topJumps[-1], len(results) - 1)][1]
     hi = results[0][1]
     norm = matplotlib.colors.Normalize(2 * low - hi, hi)
     curId = 0
+    levs = []
```

Comment:
Can we add 2 more characters and call it `levels`?

Reply:
replied in the email